src/isar_robot/robotinterface.py

```diff
@@ -102,21 +104,21 @@ def get_telemetry_publishers(self, queue: Queue, robot_id: str) -> List[Thread]:
         battery_publisher: MqttTelemetryPublisher = MqttTelemetryPublisher(
             mqtt_queue=queue,
             telemetry_method=self._get_battery_telemetry,
-            topic=f"isar/{robot_id}/battery",
+            topic=f"isar/{isar_id}/battery",
             interval=5,
             retain=False,
         )
         battery_thread: Thread = Thread(
             target=battery_publisher.run,
-            args=[robot_id],
+            args=[isar_id, robot_name],
             name="ISAR Robot Battery Publisher",
             daemon=True,
         )
         publisher_threads.append(battery_thread)
 
         return publisher_threads
 
-    def _get_pose_telemetry(self, robot_id: str) -> str:
+    def _get_pose_telemetry(self, robot_name: str, isar_id: str) -> str:
```

Comment:
```suggestion
    def _get_pose_telemetry(self, isar_id: str, robot_name: str) -> str:
```

Reply:
Everywhere else the order has been reversed.

Reply:
done